### ISSUE TITLE
Masonry breaks ONLY on iPad (iOS < 7) and only on portrait

### ISSUE THREAD

**Issue by MartinMates** [desandro/masonry#693]
Ok guys this is a tricky one. I have Masonry up and running on my full responsive website http://www.nikistrbian.com/belly-baby-faq/ . When I test it on all resolutions (in chrome dev tools) it works fine (768px wide screen included).

However when I try it on actual device iPad 4th (iOS 7). Masonry doesn't work on portrait (768px wide screen). It simply stack images one-per line. When you turn the device to landscape it works just fine. The same happens on iPad mini and all older iOS. On iPad Air (iOS 8) it works fine.

Things go wilder when you test it on Android Galaxy Tab 4 (800px wide). It works just fine both portrait and landscape mode.

I tested it out and Masonry initialize, has the right column width and all items.

My code:
```javascript
$referenceImages.imagesLoaded(function () {
        jQuery('.masonry').masonry({
            columnWidth: '.photo-reference',
            itemSelector: '.masonry-item',
            layoutPriorities: {
                shelfOrder: 5
            }
        });
    });
```

It is important to note, that I am using imagesLoaded plugin and it fires Masonry when hidden images called "referenceImages" are loaded.

I consider this as a bug since it works on any other devices and the same screen proportions in Chrome. Any ideas? :-/

Broken layout on iPad 4:
![pasted_image_28_04_15_09_11](https://cloud.githubusercontent.com/assets/8404493/7364433/a620494a-ed86-11e4-8168-811a630110ee.jpg)

The same iPad on landscape:
![pasted_image_28_04_15_09_12](https://cloud.githubusercontent.com/assets/8404493/7364443/be1618b8-ed86-11e4-8b49-e09c38d77d4c.jpg)

The same proportions as iPad in Google Chrome:
![pasted_image_28_04_15_09_15](https://cloud.githubusercontent.com/assets/8404493/7364470/2c5f87be-ed87-11e4-9bb6-f3414bac07d5.jpg)

**Comment by megao** [comment]
Ok I am having the same issue. @MartinMates Have you found a solution? Your site looks offline. Anybody else having the same problem? @desandro There is no way to recreate the scenario using chrome dev tools so I cannot figure out a way to pinpoint it.

**Comment by MartinMates** [comment]
Hi @megao ! Unfortunately I haven't found the solution. I have tried everything but no luck. My website is now live on http://www.nikistrbian.com/belly-baby-faq/ .

We agreed with the client that most users are going to have their tablets in landscape mode probably. And since its OK on new devices (iOS 8 and higher) I decided not to deal with it any more. But my webdesign perfectionist inside me is crying :-) It sucks.

Let me know if you find some clues. Good luck!
